README.md

```diff
@@ -68,6 +67,26 @@ workers implement the Remote Worker API.
 * [Scoot](https://github.com/twitter/scoot) (open source)
 * [Turbo Cache](https://github.com/allada/turbo-cache) (open source)
 
+### Workers
+Servers generally distribute work to a fleet of workers. While there is no
+standard API for communication between the server and workers, links to the
+APIs from some existing implementations are provided as a reference below.
+
+The [Remote Worker API](https://docs.google.com/document/d/1s_AzRRD2mdyktKUj2HWBn99rMg_3tcPvdjx3MPbFidU) 
+defines a generic protocol for worker and server communication, although, 
+this API is considered too heavyweight for most use-cases.
+
+*Adhering to any one of these protocols is not a requirement.*
```

Comment:
Nit: I agree with all of the sentences under this heading, but I think that the order in which they are provided can be improved. What's your thought on the following?

> Servers generally distribute work to a fleet of workers. The [Remote Worker API](https://docs.google.com/document/d/1s_AzRRD2mdyktKUj2HWBn99rMg_3tcPvdjx3MPbFidU) defines a generic protocol for worker and server communication, although, this API is considered too heavyweight for most use-cases. Because of that, many implementations have designed their own protocols. Links to these APIs are provided as a reference below. *Adhering to any one of these protocols is not a requirement.*
>
> - List of protocols goes here.

Reply:
I like it. Updated!